pilot/pkg/model/config.go

```diff
@@ -297,21 +292,11 @@ type IstioConfigStore interface {
 }
 
 const (
-	// IstioAPIGroupDomain defines API group domain of all Istio configuration resources.
-	// Group domain suffix to the proto schema's group to generate the full resource group.
-	IstioAPIGroupDomain = ".istio.io"
-
 	// Default API version of an Istio config proto message.
 	istioAPIVersion = "v1alpha2"
 
 	// NamespaceAll is a designated symbol for listing across all namespaces
 	NamespaceAll = ""
-
-	// IstioMeshGateway is the built in gateway for all sidecars
-	IstioMeshGateway = "mesh"
```

Comment:
I wouldnt move this.. this is pilot level concept. labels n such are common ones

Reply:
@rshriram This is used by the validation function for VirtualService, which has been moved to `pkg/config` ... seemed that this should be moved as well.